armi/settings/settingsIO.py

```diff
@@ -311,7 +323,13 @@ def _preprocessYaml(self, settingData):
             else:
                 cleanedData[settingObj.name] = settingDatum
 
+        # add ARMI version to the settings YAML
+        if CONF_VERSIONS not in cleanedData:
+            cleanedData[CONF_VERSIONS] = {}
+        cleanedData[CONF_VERSIONS]["armi"] = version
```

Comment:
Where does `version` come from here? Am I being a little dense?

Reply:
At the top of the file it says:

```python
from armi.meta import __version__ as version
```

This is where we store the ARMI version in `armi/meta.py`.

Reply:
I could remove the rename.  Perhaps it would have been more clear without it?

Reply:
no no I like it! I was just assuming it was something that would show up in the diff. This makes sense.